packages/mds-policy-author/tests/api.spec.ts

```diff
@@ -30,7 +30,8 @@ import db from '@mds-core/mds-db'
 import { clone } from '@mds-core/mds-utils'
 import { Policy } from '@mds-core/mds-types'
 import { ApiServer } from '@mds-core/mds-api-server'
-import { TEST1_PROVIDER_ID, TEST3_PROVIDER_ID } from '@mds-core/mds-providers'
+import { AccessTokenScope } from '@mds-core/mds-api-scopes'
```

Comment:
`mds-api-scopes` is not declared as a dependency in `package.json` or a project reference in `tsconfig.build.json`

Reply:
Thanks